setup.py

```diff
@@ -4,6 +4,7 @@
 
 # Optional dependency groups.
 extras = {
+    'ik': ['roboticstoolbox-python', 'spatialmath-python'],
```

Comment:
remove spatialmath

Reply:
we need to keep `spatialmath` per https://github.com/iamlab-cmu/isaacgym-utils/pull/29#discussion_r789678408